clerk/frontend/app/page.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+import { auth } from "@clerk/nextjs";
+import React from "react";
+import useSWR from "swr";
+
+const getUsers = async (sessionId: string) => {
+  const res = await (
+    await fetch(process.env.NEXT_PUBLIC_API_BASE_URL + "/users", {
+      credentials: "include",
+      headers: {
+        "X-CLERK-SESSION_ID": sessionId,
```

Comment:
in the backend it is `X-CLERK_SESSION_ID`

Reply:
Yes I noticed that it is already fixed it will update in the next commit